tests/e2e/repo_finder/commit_finder.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
```

Comment:
FYI in #520, we have decided to avoid running Python scripts with system Python and instead use the Python interpreter in the virtual env. With that, we opt to avoid using the Python shebang in the scripts to avoid any confusion regarding how the scripts are run.
I saw that in `integration_tests.sh` you run this script with `python <script-name>` which is good as is. However, the shebang here should be removed.